core/domain/question_jobs_one_off.py

```diff
@@ -120,6 +123,62 @@ def reduce(key, values):
         yield (key, values)
 
 
+class FixQuestionImagesStorageOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """This job is used to ensure that all question related images are present
+    in the correct storage path.
+    """
+
+    _IMAGE_COPIED = 'question_image_copied'
+    _DELETED_KEY = 'question_deleted'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [question_models.QuestionModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            yield (FixQuestionImagesStorageOneOffJob._DELETED_KEY, 1)
+            return
+
+        question = question_services.get_question_by_id(item.id, strict=False)
```

Comment:
Can't you just call the get_question_from_model function, since `item` is the QuestionModel?

Reply:
Oops, you're right. Done.

Reply:
I think you made the change somewhere else and not here (anyway, thanks for that :P)

Reply:
Ah sorry, done.